src/Files.Uwp/UserControls/MultitaskingControl/TabItem/TabItem.cs

```diff
@@ -29,6 +29,14 @@ public string Description
             set => SetProperty(ref description, value);
         }
 
+        private string hoverDisplayText;
+
+        public string HoverDisplayText
```

Comment:
Not a requirement but we're trying to add proper comments for new properties to better define what they're used for.

Reply:
Alright, I will add this in a new commit!

Reply:
It has been added. Was it the expected format?

Reply:
Perfect